.pre-commit-config.yaml

```diff
@@ -77,6 +76,12 @@ repos:
     hooks:
       - id: shellcheck
 
+  - repo: https://github.com/kynan/nbstripout
+    rev: 0.6.1
+    hooks:
+      - id: nbstripout
+        args: [--extra-keys=metadata.kernelspec metadata.language_info.version]
+
```

Comment:
This pre-commit clears and strip out metadata for the jupyter notebooks so that Jupyter Book can render it dynamically.